packages/node/src/init.ts

```diff
@@ -9,13 +9,15 @@ import { ApiService } from './indexer/api.service';
 import { FetchService } from './indexer/fetch.service';
 import { ProjectService } from './indexer/project.service';
 import { yargsOptions } from './yargs';
+const pjson = require('../package.json');
 
 const { argv } = yargsOptions;
 
 const DEFAULT_PORT = 3000;
 const logger = getLogger('subql-node');
 
 export async function bootstrap() {
+  logger.info(`Current subql-node version is ${pjson.version}`);
```

Comment:
Then we display the correct package name for each sdk

```suggestion
  logger.info(`Current ${pjson.name} version is ${pjson.version}`);
```